github/issues.go

```diff
@@ -36,6 +36,7 @@ type Issue struct {
 	Milestone        *Milestone        `json:"milestone,omitempty"`
 	PullRequestLinks *PullRequestLinks `json:"pull_request,omitempty"`
 	Repository       *Repository       `json:"repository,omitempty"`
+	Reactions        *ReactionsSummary `json:"reactions,omitempty"`
```

Comment:
I didn't want any confused with `type Reaction struct {...}`.

In the GitHub API docs, whenever you see the "`reactions` object", the text describes them as "Reactions summary", so I felt this was appropriate.

@willnorris, what do you think?